src/Lidar.cc

```diff
@@ -192,6 +192,26 @@ bool Lidar::Update(const ignition::common::Time &/*_now*/)
   return false;
 }
 
+//////////////////////////////////////////////////
+void Lidar::ApplyNoise()
+{
+  if (this->dataPtr->noises.find(LIDAR_NOISE) != this->dataPtr->noises.end())
+  {
+    for (unsigned int j = 0; j < this->VerticalRayCount(); ++j)
+    {
+      for (unsigned int i = 0; i < this->RayCount(); ++i)
+      {
+        int index = j * this->RangeCount() + i;
```

Comment:
one more `this->RangeCount()` to `this->RayCount()` here

Reply:
5d3be6d3c970a1827a5cdaf60e8a9e92e48a1558